tests/images/debian-systemd/files/bootstrap.sh

```diff
@@ -595,8 +595,8 @@ install_tedge() {
 
     # Check if any packages are incompatible, if so remove the previous version first
     # Use new and old packages names, and check each of them one by one
-    packages="tedge tedge-mapper tedge-agent tedge-log-plugin c8y-configuration-plugin c8y-remote-access-plugin"
-    packages="$packages tedge tedge_mapper tedge_agent c8y_log_plugin c8y-log-plugin c8y_configuration_plugin"
+    packages="tedge tedge-mapper tedge-agent tedge-log-plugin tedge-configuration-plugin c8y-remote-access-plugin"
+    packages="$packages tedge tedge_mapper tedge_agent c8y_log_plugin c8y-log-plugin c8y_configuration_plugin c8y-configuration-plugin"
```

Comment:
Out of scope of this PR, why 'tedge' is added twice?

Reply:
Looks to be an oversight...though it will just be a no-op if the package does not exist anyway, but yes it would be good to clean it up.